assembl/static2/package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "dev": "webpack -d --watch",
-    "build": "webpack -p",
+    "dev": "BABEL_ENV=dev webpack -d --watch",
```

Comment:
I thought -d for webpack meant NODE_ENV=development and -p means NODE_ENV=production. 
This is not the case at all
```  -d                                    shortcut for --debug --devtool sourcemap --output-pathinfo
  -p                                    shortcut for --optimize-minimize

https://babeljs.io/docs/usage/babelrc/ says 

> The env key will be taken from process.env.BABEL_ENV, when this is not available then it uses process.env.NODE_ENV if even that is not available then it defaults to "development".

So I think you can simplify the configuration here, don't use BABEL_ENV, don't do the change in supervisorctl.tmpl and just modify the build command in package to be:
NODE_ENV=production webpack -p
This is actually what I do in another project.
NODE_ENV=production webpack --config webpack.config.prod.js
 So it seems you already have